src/renderer/js/component/shapeShift/internal/form.jsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import Link from "component/link";
+import { getExampleAddress } from "util/shape_shift";
+import { Submit, FormRow } from "component/form";
+import type { ShapeShiftFormValues, Dispatch } from "redux/actions/shape_shift";
+
+type ShapeShiftFormErrors = {
+  returnAddress?: string,
+};
+
+type Props = {
+  values: ShapeShiftFormValues,
+  errors: ShapeShiftFormErrors,
+  touched: boolean,
+  handleChange: Event => any,
+  handleBlur: Event => any,
+  handleSubmit: Event => any,
+  isSubmitting: boolean,
+  shiftSupportedCoins: Array<string>,
+  originCoin: string,
+  updating: boolean,
+  getCoinStats: Dispatch,
+  receiveAddress: string,
+  originCoinDepositFee: number,
+  originCoinDepositMin: string,
+  originCoinDepositMax: number,
+};
+
+export default (props: Props) => {
+  const {
+    values,
+    errors,
+    touched,
+    handleChange,
+    handleBlur,
+    handleSubmit,
+    isSubmitting,
+    shiftSupportedCoins,
+    originCoin,
+    updating,
+    getCoinStats,
+    receiveAddress,
+    originCoinDepositMax,
+    originCoinDepositMin,
+    originCoinDepositFee,
+  } = props;
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="form-field">
+        <span>{__("Exchange")} </span>
+        <select
+          className="form-field__input form-field__input-select"
+          name="originCoin"
+          onChange={e => {
+            getCoinStats(e.target.value);
+            handleChange(e);
+          }}
+        >
+          {shiftSupportedCoins.map(coin => (
+            <option key={coin} value={coin}>
+              {coin}
+            </option>
+          ))}
+        </select>
+        <span> {__("for LBC")}</span>
+        <p className="shapeshift__tx-info help">
+          {!updating &&
```

Comment:
1) Is it possible to display the actual rate here?
2) This can be more compact. If we have the rate, proposed text is "Receive X LBC per Y BTC, less Z LBC fee. Max: W BTC, Min: V BTC"
3) Typo in minimum

Reply:
Yes we can display the actual rate. I'll add that